Gemfile

```diff
@@ -4,7 +4,7 @@ source 'https://rubygems.org'
 
 # gem "rails"
 
-ruby "3.1.2"
+ruby '3.1.2'
```

Comment:
Looks like the build failed because of a Ruby version mismatch.  Seems like it should be '3.1.0'.

Reply:
Thanks for pointing that out! I edited the version in the GitHub workflows ruby.yml file to match the project version.

Reply:
I just saw that the ubuntu version specified in .github/workflows/ruby.yml doesn't support Ruby 3.1.2. I'm working on changing the project Ruby version to 3.1.0.